test/clojure-mode-font-lock-test.el

```diff
@@ -169,6 +169,9 @@ POS."
 
 (ert-deftest clojure-mode-syntax-table/namespaced-symbol ()
   :tags '(fontification syntax-table)
+  (should (eq (clojure-test-face-at 1 5 "one.p") 'font-lock-type-face))
```

Comment:
This seems unrelated to the rest of the test.

Reply:
I'm closing this PR and going to send you a new PR where I fixed this point.